extra/Dockerfile.armv7-armhf

```diff
@@ -36,7 +36,11 @@ RUN apt-get update && \
     python-wheel \
     tzdata \
     v4l-utils \
-    zlib1g-dev && \
+    zlib1g-dev \
+    build-essential \
```

Comment:
Please keep the contents in Docker images minimal. Docker images are meant for runtime not for development. Feel free to install additional tools, when starting your container.

Reply:
Found it. `pip install lxml` needs the additional time. But where is it installed? I don't see commands for that. Am I blind?

Reply:
lxml is a dependency of webdavclient3. webdavclient3 is defined as a requirement in setup.py.
The apt packages which I added to the arm Dockerfile are actually needed to successfully compile lxml. The ubuntu Dockerfile seems to not have this issue.

Reply:
I added a pre stage docker container to the arm dockerfile which is only for building the modules. Later only the finished python wheels are copied to the production container. That removes the necessity of installing packages only needed for building inside the production docker container. Please have a look. After all the new arm container grows with this PR only 1MB (from 375M to 376M)